_alerts/2015-07-04-spv-mining.html

```diff
@@ -9,7 +9,7 @@
 update: 2015-07-06 02:00.  All times are UTC.</em></p>
 
 **Note: this alert is on-going: the situation has not yet been
-resolved. ([Update #1](#update-1))**
+resolved. ([Read more](#cause))**
```

Comment:
Please feel free to ignore it, but FWIW, I didn't notice the paragraph due to (1) it being at the bottom of the page and (2) it not using any title, nothing very obvious.